airflow/dags/select_top5_and_save_dag.py

```
@@ -0,0 +1,75 @@
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from datetime import timedelta
+import pendulum
+
+tz = pendulum.timezone("Asia/Seoul")
+
+# 1) Articles에서 Top5 선정
+def task_select_top5_articles(**context):
```

Ruff (0.12.2)

Unused function argument: `context`

(ARG001)